performances/performance_test/include/performance_test/ros2/executors.hpp

```diff
@@ -28,6 +28,12 @@ enum ExecutorType
     STATIC_SINGLE_THREADED_EXECUTOR = 2,
 };
 
+enum NodeType
```

Comment:
You should probably either rename this file or move the node utilities that you created somewhere else

Reply:
Right, I didn't know where to put it at the moment